### PR TITLE
converted all array() calls to [], changed some formatting, etc.

### PR DESCRIPTION
**Pull request by naneri**
* Converted all array() calls to [], to provide consistency across the file, as both array() and [] have been used. 
* added dockblocks to methods and properties to describe what methods/properties 
* formatted the code for readability

### CODE REVIEW

File: LSTelegramNotify.php [hunk 1]
```diff
@@ -78,53 +103,61 @@ public function sendMessage($surveyId, $text)
                     $this->get('ChatId') // Global
                 ) . 
                 '&text=' . urlencode($text);
+
         file_get_contents($url);
     }
 
+    /**
+     *
```
Comment (vitormattos) [inline LSTelegramNotify.php]: Empty? I suggest to put a description about this event for follow the pr objective or maintain without this docblock. Also is possible put annotation of type of return.

File: LSTelegramNotify.php [hunk 2]
```diff
@@ -78,53 +103,61 @@ public function sendMessage($surveyId, $text)
...
-                )
-            ));
+                ]
+            ]
+        ]);
 
     }
 
+    /**
+     *
```
Comment (vitormattos) [inline LSTelegramNotify.php]: the same of before.

**Review by vitormattos — CHANGES_REQUESTED** [review]
Congratulations for your contribution!

Here are some suggestions to walk towards the acceptance of the PR

Do you know PHPCS? Is a good moment to put in this project validating the source using PSR12 and determine the style pattern of code.

One suggestion is to put the command to run PHPCS in script block of composer.json like this:  https://github.com/LyseonTech/dou-collector/blob/master/composer.json#L28

**Comment by naneri** [comment]
@vitormattos concerning the Method descriptions - I am not 100% sure that I would describe those correctly. Also dockblocks did not generate any return type, because none of the methods returns anythings

**Comment by vitormattos** [comment]
To use a pattern of docblock, I recomend to install [psalm](https://psalm.dev/) OR [Phan](https://github.com/phan/phan) in this PR

To define pattern of coding, I recomend [PHPCS](https://github.com/squizlabs/PHP_CodeSniffer) using PSR12.

Different people may want different standards. Using tools to define industry standards is the best way.

**Comment by vitormattos** [comment]
Good! You will implement PHPCS or another static analyses tool in this PR?

**Comment by naneri** [comment]
@vitormattos I have PSR-12 code style as a preset in PHPStorm, though it seems like very different to the way the repo was before the pull requests. Concerning adding a static analyzer in this PR, was not planning to do that.

File: LSTelegramNotify.php (hunk 1 above)
Comment (naneri) [reply LSTelegramNotify.php]: added void return type

**Comment by vitormattos** [comment]
very good contribution

**Pull request merged**